### DISCUSSION TITLE
Plugin to track cost of supplies while killing bosses

### DISCUSSION THREAD

**Discussion by Gnosticz** [runelite/runelite#15351] [Ideas]
I wanted to know how much more it would cost me to run tome of fire and fire surge instead of Trident while killing Zulrah, but keeping track of the cost of supplies would be a pain. This plugin could be great for determining the best gear to kill different bosses.

**Comment by LlemonDuck** [comment]
There is a very popular hub plugin called [Supplies Tracker](https://runelite.net/plugin-hub/show/supplies-tracker) that fills this purpose.
